service/pom.xml

```diff
@@ -400,35 +397,15 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <version>3.0.0</version>
-                <executions>
-                    <execution>
-                        <id>enforce-banned-dependencies</id>
-                        <goals>
-                            <goal>enforce</goal>
-                        </goals>
-                        <configuration>
-                            <rules>
-                                <bannedDependencies>
-                                    <excludes>
```

Comment:
Even though these microservices do not use these dependencies, someone may add them again in the future by accident....why not leave these rules in place?

Reply:
they shouldn't have been added here.  This exact configuration is in the parent pom (i.e. datawave-microservice-service-parent)